### ISSUE TITLE
`tar` protocol does not take following `ssh` into account

### ISSUE THREAD

**Issue by mraspaud** [fsspec/filesystem_spec#946]
It looks like that when I am trying to open a tar file on a remote (ssh) system, the tar protocol does not use remote access and instead wants to read a local file:

```python
import fsspec
fsspec.open_files("tar::ssh://myserver/tmp/file.tar")
```

crashes with a `FileNotFoundError`, while

```python
from fsspec.implementations.tar import TarFileSystem
tfs = TarFileSystem("ssh://myserver/tmp/file.tar")
```
works fine... Am I doing something wrong?

**Comment by mraspaud** [comment]
@martindurant  Thanks for the quick fix! however, I installed fsspec from master, and I still get the error...

**Comment by mraspaud** [comment]
the fix in #950 works for my case.
